src/components/20-molecules/datepicker/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 7.0.1
+
+- Remove datepicker specific stylings from axa dropdown. (#1720)
```

Comment:
IMHO we don't need a changelog for internal refactorings. We also did not write changelog for all the releases we did, for instance, when someone is doing a global release.

Reply:
In this case there were actually changes in the dropdown component

Reply:
If you changed something at datepicker code i would say that should be part of the changelog, even if its internal. So we can also read the changelog to know what we have doing in past. But the changed need to be big enough to describe them.
It would be more interesting if you describe what have changed for the costumers. Is that fix visible somewhere? Or do you just use the new datepicker in that release?